src/header.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/sh
+
+set -eu
+
+TEMPDIR=`mktemp -d`
+PREFIX="env"
+FORCE=0
+INSTALLER="rattler"  # Default to rattler
+CREATE_ACTIVATION_SCRIPT=false
+
+# Pixi Constants ./lib.rs
+PIXI_PACK_CHANNEL_DIRECTORY=""
+PIXI_PACK_METADATA_PATH=""
+PIXI_PACK_DEFAULT_VERSION=""
+
+USAGE="
+usage: $0 [options]
+
+Unpacks an environment packed with pixi-pack
+
+-f           no error if environment already exists
+-h           print this help message and exit
+-p ENV       environment prefix, defaults to $PREFIX
+-i INSTALLER create the environment using the specified installer defaulting to $INSTALLER
+-a           create an activation script to activate the environment
+"
+
+while getopts ":fhai:p:" x; do
+    case "$x" in
+        f)
+            FORCE=1
+            ;;
+        p)
+            PREFIX="$OPTARG"
+            ;;
+        i)
+            INSTALLER="$OPTARG"
```

Comment:
i don't think we need different installers, let's only support our own pixi-pack one that we directly ship in the binary